Dockerfile

```diff
@@ -4,22 +4,23 @@ ARG ROSETTA_DEVNET_TAG=v0.3.5
 ARG ROSETTA_MAINNET_TAG=v0.3.5
 ARG ROSETTA_DOCKER_SCRIPTS_TAG=v0.2.3
 
-# Corresponds to elrond-go v1.3.50-hf01
+# Corresponds to mx-chain-go v1.3.50-hf01
 ARG CONFIG_DEVNET_TAG=D1.3.50.0-hf01
-# Corresponds to elrond-go v1.3.50
+# Corresponds to mx-chain-go v1.3.50
 ARG CONFIG_MAINNET_TAG=v1.3.50.0
 
 # Clone repositories
 WORKDIR /repos
-RUN git clone https://github.com/ElrondNetwork/rosetta-docker-scripts.git --branch=${ROSETTA_DOCKER_SCRIPTS_TAG} --single-branch --depth=1
-RUN git clone https://github.com/ElrondNetwork/elrond-config-devnet --branch=${CONFIG_DEVNET_TAG} --single-branch --depth=1
-RUN git clone https://github.com/ElrondNetwork/elrond-config-mainnet --branch=${CONFIG_MAINNET_TAG} --single-branch --depth=1
+RUN git clone https://github.com/multiversx/mx-chain-rosetta-docker-scripts.git --branch=${ROSETTA_DOCKER_SCRIPTS_TAG} --single-branch --depth=1
+RUN git clone https://github.com/multiversx/mx-chain-devnet-config --branch=${CONFIG_DEVNET_TAG} --single-branch --depth=1
+# TODO: Adjust URL upon migration
```

Comment:
remove TODO and replace L17 with `https://github.com/multiversx/mx-chain-mainnet-config` ?

Reply:
Fixed.